xcat-inventory/xcclient/inventory/manager.py

```diff
@@ -330,6 +330,13 @@ def validate_args(args, action):
             if not os.path.isdir(mydir):
                 raise CommandException("Error: the directory %(f)s does not exist or is not a directory!", f=mydir)
        
+    if args.exclude:
+        if args.type or args.name:
+            raise CommandException("Error: \"-c|--exclude\" cannot be used with \"-t|--type\" or \"-o|--objects\"") 
```

Comment:
Why we add this limitation? Maybe this is allowed: ` -t a,b,c --exclude a `, just useful when scripting using xcat-inventory.

And if you need it, there should be `-x', instead of `-c`

Reply:
fixed